addons/web/static/src/views/fields/color_picker/color_picker_field.js

```diff
@@ -14,7 +14,7 @@ export class ColorPickerField extends Component {
         canToggle: { type: Boolean },
     };
 
-    static RECORD_COLORS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11];
+    static RECORD_COLORS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
```

Comment:
The `colors` props of `ColorList` is annoying. Shouldn't we remove it? I don't see the point of generating those arrays... which are basically the indices of colors defined by ColorList.